inbox/models/backends/gmail.py

```diff
@@ -25,33 +25,18 @@
 GOOGLE_EMAIL_SCOPE = "https://mail.google.com/"
 GOOGLE_CONTACTS_SCOPE = "https://www.google.com/m8/feeds"
 
-OAUTH_CLIENT_ID = config.get_required("GOOGLE_OAUTH_CLIENT_ID")
-OAUTH_CLIENT_SECRET = config.get_required("GOOGLE_OAUTH_CLIENT_SECRET")
-OAUTH_REDIRECT_URI = config.get_required("GOOGLE_OAUTH_REDIRECT_URI")
-
 
 class GmailAccount(OAuthAccount, ImapAccount):
+    OAUTH_CLIENT_ID = config.get_required("GOOGLE_OAUTH_CLIENT_ID")
+    OAUTH_CLIENT_SECRET = config.get_required("GOOGLE_OAUTH_CLIENT_SECRET")
+
     id = Column(ForeignKey(ImapAccount.id, ondelete="CASCADE"), primary_key=True)
 
     __mapper_args__ = {"polymorphic_identity": "gmailaccount"}
 
     client_id = Column(String(256))
     scope = Column(String(512))
 
-    # XXX: These fields are not currently used.
```

Comment:
weren't these already dropped? https://github.com/closeio/sync-engine/pull/28

Reply:
I just changed base for this PR. Reload please.